var/spack/repos/builtin/packages/r/package.py

```diff
@@ -63,7 +64,7 @@ class R(Package):
     depends_on('ncurses')
     depends_on('icu4c')
     depends_on('glib')
-    depends_on('zlib@:1.2.8')
+    depends_on('zlib@1.2.5:1.2.9')
```

Comment:
zlib@1.2.9 apparently had a fatal bug and shouldn't be used.  This bit of the diff is obsoleted by #2901 (which also ignores zlib@1.2.9's foibles...).